### DISCUSSION TITLE
How to get the .uf2 file

### DISCUSSION THREAD

**Discussion by BernardBaron** [earlephilhower/arduino-pico#2567] [Q&A] [answered]
How can i get the .uf2 file and save it on my local pc?. So if i want to upload it on another board i just copy paste the file. i am using arduino IDE

**Comment by Andy2No** [comment]
In your preferences.txt file for Arduino (File->Preferences then click on the path for preferences.txt, near the bottom), add a line like this:

build.path=C:\arduino-output

Exit the Arduino IDE before saving that file, or it will get overwritten.

Your build files are then always saved into that folder. Copy what you want to keep into another one.

**Comment by maxgerhardt** [comment, marked as answer]
![grafik](https://github.com/user-attachments/assets/482b2816-1a84-4807-b676-ad9b110fa519)
then
![grafik](https://github.com/user-attachments/assets/7ed9734a-1242-4a67-b125-fc64fe2df3c5)
![grafik](https://github.com/user-attachments/assets/c59e555f-d74e-4f68-821c-949f62521932)
![grafik](https://github.com/user-attachments/assets/79ab8bdb-78c7-4536-9b01-734349baa6ed)
